### YamlMime:UniversalReference
ms.openlocfilehash: 80639ea90d12b3e57b99dc309a62df521d35f0f5
ms.sourcegitcommit: efa2d98deffe8a0d41a8d63f9f07aa720862e6ab
ms.translationtype: MT
ms.contentlocale: zh-TW
ms.lasthandoff: 12/13/2018
ms.locfileid: "52032693"
items:
- uid: azure-servicefabric.ValidationFailedChaosEvent
  name: ValidationFailedChaosEvent
  fullName: ValidationFailedChaosEvent
  children:
  - azure-servicefabric.ValidationFailedChaosEvent.reason
  langs:
  - typeScript
  type: interface
  summary: <span data-ttu-id="0e84b-101">在驗證期間的失敗對應的混亂事件。</span><span class="sxs-lookup"><span data-stu-id="0e84b-101">Chaos event corresponding to a failure during validation.</span></span>
  extends:
    name: azure-servicefabric.ChaosEvent
  package: azure-servicefabric
- uid: azure-servicefabric.ValidationFailedChaosEvent.reason
  name: reason
  fullName: reason
  children: []
  langs:
  - typeScript
  type: property
  summary: <span data-ttu-id="0e84b-102">描述為何產生 ValidationFailedChaosEvent。</span><span class="sxs-lookup"><span data-stu-id="0e84b-102">Describes why the ValidationFailedChaosEvent was generated.</span></span> <span data-ttu-id="0e84b-103">這可能是因為多個 MaxPercentUnhealthyNodes MaxClusterStabilizationTimeout 超過狀況不良。</span><span class="sxs-lookup"><span data-stu-id="0e84b-103">This may happen because more than MaxPercentUnhealthyNodes are unhealthy for more than MaxClusterStabilizationTimeout.</span></span> <span data-ttu-id="0e84b-104">做為字串，因此將處於 ValidationFailedChaosEvent 的原因與 Reason 屬性。</span><span class="sxs-lookup"><span data-stu-id="0e84b-104">This reason will be in the Reason property of the ValidationFailedChaosEvent as a string.</span></span>
  optional: true
  syntax:
    content: 'reason?: string'
    return:
      type:
      - string
  package: azure-servicefabric
references:
- uid: azure-servicefabric.ChaosEvent
  name: ChaosEvent
  spec.typeScript:
  - name: ChaosEvent
    fullName: ChaosEvent
    uid: azure-servicefabric.ChaosEvent
